scripts/src/java/org/oppia/android/scripts/common/ScriptExemptions.kt

```diff
@@ -0,0 +1,632 @@
+    "WalkthroughFragmentChangeListener.kt",
+    "WalkthroughViewModel.kt",
+    "WalkthroughActivityPresenter.kt",
+    "WalkthroughActivityListener.kt",
+    "WalkthroughPageChanger.kt",
+    "WalkthroughTopicViewModel.kt",
+    "WalkthroughTopicSummaryViewModel.kt",
+    "WalkthroughTopicHeaderViewModel.kt",
+    "WalkthroughTopicListFragmentPresenter.kt",
+    "WalkthroughTopicItemViewModel.kt",
+    "WalkthroughFinalViewModel.kt",
+    "WalkthroughFinalFragmentPresenter.kt",
+    "WalkthroughFinalListener.kt",
+    "WalkthroughEndPageChanger.kt",
+    "ProfileEditDialogInterface.kt",
+    "ProfileEditFragment.kt",
+    "ProfileEditViewModel.kt",
+    "ProfileResetPinActivityPresenter.kt",
+    "ProfileEditActivityPresenter.kt",
+    "ProfileListFragmentPresenter.kt",
+    "ProfileListActivityPresenter.kt",
+    "ProfileListViewModel.kt",
+    "ProfileEditDeletionDialogFragment.kt",
+    "ProfileEditFragmentPresenter.kt",
+    "ProfileRenameActivityPresenter.kt",
+    "ProfileResetPinViewModel.kt",
+    "ProfileRenameViewModel.kt",
+    "RecyclerViewMatcher.kt",
+    "EspressoTestsMatchers.kt",
+    "ProgressMatcher.kt",
+    "OrientationChangeAction.kt",
+    "FontSizeMatcher.kt",
+    "DragViewAction.kt",
+    "DrawableMatcher.kt",
+    "TabMatcher.kt",
+    "ClickActions.kt",
+    "Constants.kt",
+    "OppiaRetrofit.kt",
+    "NetworkSettings.kt",
+    "GaeState.kt",
+    "GaeExplorationContainer.kt",
+    "GaeSolution.kt",
+    "GaeUserSuppliedFeedback.kt",
+    "GaeStorySummary.kt",
+    "GaeStory.kt",
+    "GaeStoryNode.kt",
+    "GaeWrittenTranslation.kt",
+    "GaeExpSummary.kt",
+    "GaeSubtitledHtml.kt",
+    "GaeInteractionInstance.kt",
+    "GaeStateClassifier.kt",
+    "GaeFeedbackReportingSystemContext.kt",
+    "GaeRuleSpec.kt",
+    "GaeQuestionPlayer.kt",
+    "GaeRecordedVoiceovers.kt",
+    "GaeTopic.kt",
+    "GaeConceptCard.kt",
+    "GaeQuestion.kt",
+    "GaeVoiceover.kt",
+    "GaeTopicSummary.kt",
+    "GaeParamChange.kt",
+    "GaeClassroom.kt",
+    "GaeAnswerGroup.kt",
+    "GaeFeedbackReport.kt",
+    "GaeExploration.kt",
+    "GaeSubtopic.kt",
+    "GaeParamSpec.kt",
+    "GaeSkillContents.kt",
+    "GaeSubtopicPageContents.kt",
+    "GaeWrittenTranslations.kt",
+    "GaeFeedbackReportingAppContext.kt",
+    "GaeFeedbackReportingDeviceContext.kt",
+    "GaeCustomizationArgs.kt",
+    "GaeOutcome.kt",
+    "GaeFeedbackReportingEntryPoint.kt",
+    "GaeHint.kt",
+    "GaeSubtopicSummary.kt",
+    "NetworkApiKey.kt",
+    "NetworkModule.kt",
+    "ScriptExemptions.kt",
+    "ScriptResultConstants.kt",
+    "AccessibilityTestRule.kt",
+    "DisableAccessibilityChecks.kt",
+    "DeveloperOptionsFragment.kt",
+    "DeveloperOptionsActivityPresenter.kt",
+    "DeveloperOptionsStarterModule.kt",
+    "DeveloperOptionsStarter.kt",
+    "DeveloperOptionsStarterImpl.kt",
+    "DeveloperOptionsFragmentPresenter.kt",
+    "DeveloperOptionsModule.kt",
+    "CommandResult.kt",
+    "CommandExecutor.kt",
```

Comment:
Are these still needed?

Reply:
Will remove them, once I take a merge from #3340. so that, the checks won't fal.

Reply:
I have taken a merge but looks like we don't have test files: `CommandResultTest` and `CommandExecutorTest`. Hence, these are still needed to be added.

Reply:
Issue filed: https://github.com/oppia/oppia-android/issues/3436